### PR TITLE
Otelcol config update with batch size and source processor

### PR DESCRIPTION
**Pull request by pmm-sumo**
###### Description

Adds a maximum batch size limit and leverages new tag structure

###### Testing performed

- [ ] ci/build.sh
- [x] Redeploy fluentd and fluentd-events pods
- [x] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
I propose to write tests to otelcol, as it is going to be more complex with every change :)

**Comment by pmm-sumo** [comment]
Really good idea from @sumo-drosiek on the tests. Learnt a few things about Helm that way and actually found two bugs in the original proposal. Thanks for the suggestion!

**Comment by perk-sumo** [comment]
@pmm-sumo tests don't pass in Vagrant 🤔

**Comment by pmm-sumo** [comment]
> @pmm-sumo tests don't pass in Vagrant 🤔

Is it good now?

File: tests/tracing/run.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+readonly SCRIPT_PATH="$( dirname $(realpath ${0}) )"
+readonly STATICS_PATH="${SCRIPT_PATH}/static"
+readonly INPUT_FILES="$(ls "${STATICS_PATH}" | grep input)"
+readonly OUT="new_values.yaml"
+
+#docker run --rm \
+#  -v ${SCRIPT_PATH}/../../deploy/helm/sumologic:/chart \
+#  sumologic/kubernetes-tools:master \
+#  helm dependency update /chart
```
Comment (perk-sumo) [inline tests/tracing/run.sh:15]: This should be uncommented, rith?

**Pull request merged**